src/passes/Asyncify.cpp

```diff
@@ -1008,17 +1031,34 @@ struct AsyncifyLocals : public WalkerPass<PostWalker<AsyncifyLocals>> {
   AsyncifyLocals(ModuleAnalyzer* analyzer) : analyzer(analyzer) {}
 
   void visitCall(Call* curr) {
+    // Check if this is one of our fake calls.
+    if (analyzer->fakeCalls.getSetter(curr->target)) {
```

Comment:
I don't understand why this is in an `if`. If the target is not in the map, `getSetter` will throw an exception, not return an error value.

Reply:
`getSetter` of a function name will return the function, or nullptr if it's not a setter. (`getSetter` of a type is guaranteed to succeed, though.)

Reply:
Oh, right you are. I was looking at the wrong overload 😅